### PR TITLE
meson:include phosphor-dbus-interfaces dependency

### PR DESCRIPTION
**Pull request by PriyangaRamasamy**
This commit includes phosphor-dbus-interfaces as dependency module in vpd binaries.

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -25,6 +25,7 @@ endif
 build_tests = get_option('tests')
 
 sdbusplus = dependency('sdbusplus', fallback: [ 'sdbusplus', 'sdbusplus_dep' ])
+phosphor_dbus_interfaces = dependency('phosphor-dbus-interfaces')
```
Comment (jinuthomas) [inline meson.build:28]: why is this needed ? 
Could you please explain the reason for this being a standalone commit.

Comment (PriyangaRamasamy) [reply meson.build:28]: this is required to throw xyz common device errors.

this is a standalone piece as it doesn't create any dependency with other commits. secondly i need this piece in both read and write. so instead of duplicating it in multiple PRs, I created this as a standalone commit.

it is same like how I have PRs for util apis, where its usage is in some other upcoming PR.

Comment (SunnySrivastava1984) [reply meson.build:28]: This is not the correct way of splitting the commits, some usability should be there. What can be needed in future should be added along with its use.
For utility API it is still ok as it adds a functionality but adding a dependency without actually showcasing why it is being added does not make any sense.

**Review by SunnySrivastava1984 — CHANGES_REQUESTED** [review]
Should be added when required.
